### ISSUE TITLE
map not showing anything

### ISSUE THREAD

**Issue by shubham9987** [AHAAAAAAA/PokemonGo-Map#2834]
hey it not showing anything thing pls fix it fast.
thank you

**Comment by RowanF1** [comment]
You've got a very clear sentense up there, that's really kind of you. What is your problem?

**Comment by shubham9987** [comment]
https://pokemongo-mapss.herokuapp.com/
![bandicam 0015](https://cloud.githubusercontent.com/assets/19153689/17276150/7da343e2-573d-11e6-984c-14ce49af43c4.jpg)

**Comment by shubham9987** [comment]
hello rowan thank you for your quick reply
Here is my problem

**Comment by RowanF1** [comment]
Put "-sd 5" in your command line and restart, without the ".

**Comment by shubham9987** [comment]
i am using this by heroku https://pokemongo-mapss.herokuapp.com/ can you pls tell me how can i do this on heroku?

**Comment by alex-ception** [comment]
Go on the heroku interface, chose your app, go to settings and then "reveal config" you'll be able to edit conf and add it. But I'll have to warn you, it seems that an update occured today on the API, you won't be able to show pokemon for now. I'm pretty sure you have to wait for a fix.

**Comment by shubham9987** [comment]
thank you alex for reply 
but it not working

**Comment by MRokas** [comment]
@alex-ception `-sd 5` is enough. However mere 10 steps takes 1355 seconds..

Only solution now is to use very small -st (Max should be 5 imo), and workers.

**Comment by torres01** [comment]
where do i put "-sd 5" in? the config vars don't let me add another key/value

**Comment by MRokas** [comment]
`scan-delay: 5` in cofig

**Comment by shubham9987** [comment]
thank you to all for the information but still it not working

**Comment by torres01** [comment]
the "add" button is grey. can't add another key

**Comment by Hellls** [comment]
Hi guys,

Sorry to bother you but about 20 mins ago everything was working. I installed the updated version by copying and replacing the files. Now when I enter the commands with my location I get this... 


![capture](https://cloud.githubusercontent.com/assets/20753116/17276260/09cd591a-5767-11e6-84de-a1f621b73479.JPG)


I actually wasn't getting this earlier at all.

**Comment by MRokas** [comment]
`pip install -r requirements.txt`

**Comment by shubham9987** [comment]
yeah 1 hour ago my heroku app also working properly but now it showing nothing

**Comment by shubham9987** [comment]
you can also check    https://pokemongomapss.herokuapp.com/

**Comment by Hellls** [comment]
Thank you MRokas. You sir/mam are a champion!

**Comment by alex-ception** [comment]
AFAIK, It's not only heroku which is down. Game client has been updated, and so is the API I think.

**Comment by remcotroost** [comment]
Due to Niantic blocking AWS IP's, Heroku does not work anymore. Only option is to run this locally or on your own non-AWS VPS

**Comment by alex-ception** [comment]
Hum local instance does not work either...

**Comment by neobithia** [comment]
Yea, local server doesn't work for me either :(

**Comment by trusk89** [comment]
So what's the problem in the end?
